### ISSUE TITLE
Finetune the pre-trained model

### ISSUE THREAD

**Issue by mayankagrawal93** [finiteautomata/pysentimiento#1]
I was hoping could you share with us fine-tuning script for the model? I have a dataset of 50k spanish records with 10 sentiment labels. I wanted to finetune the model with it.

**Comment by finiteautomata** [comment]
Hi @mayankagrawal93. The training code was in a notebook, but now I've moved everything to a python script. Please check [bin/train.py](https://github.com/finiteautomata/pysentimiento/blob/master/bin/train.py) and look at the instructions at the `README.md`.

Feel free to ask any questions you might have
